frontend/scss/app/components/_toastr.scss

```
@@ -28,6 +30,33 @@
+    opacity: 0.45;
+    animation-name: tp__toast-progress;
+    animation-timing-function: linear;
+    animation-fill-mode: forwards;
+  }
+
+  &:hover &__progress {
+    animation-play-state: paused;
+  }
+}
+
+@keyframes tp__toast-progress {
```

Stylelint (17.12.0)

[error] Expected keyframe name "tp__toast-progress" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)